ObjcVoiceQuickstart/ViewController.m

```diff
@@ -220,23 +220,17 @@ - (BOOL)textFieldShouldReturn:(UITextField *)textField {
 
 #pragma mark - PushKitEventDelegate
 - (void)credentialsUpdated:(PKPushCredentials *)credentials {
-    const char *tokenBytes = [credentials.token bytes];
-    NSMutableString *deviceTokenString = [NSMutableString string];
-    for (NSUInteger i = 0; i < [credentials.token length]; ++i) {
-        [deviceTokenString appendFormat:@"%02.2hhx", tokenBytes[i]];
-    }
-
     NSString *accessToken = [self fetchAccessToken];
-    
-    NSString *cachedDeviceToken = [[NSUserDefaults standardUserDefaults] objectForKey:kCachedDeviceToken];
-    if (![cachedDeviceToken isEqualToString:deviceTokenString]) {
-        cachedDeviceToken = deviceTokenString;
+
+    NSData *cachedDeviceTokenData = [[NSUserDefaults standardUserDefaults] objectForKey:kCachedDeviceTokenData];
```

Comment:
I am in an opinion of call it `deviceToken` and `cachedDeviceToken`. For developers it's still just token.